packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -182,12 +182,22 @@ class SkiaGoldClient {
     final io.ProcessResult result = await process.run(imgtestCommand);
 
     if (result.exitCode != 0) {
```

Comment:
@kjlubick can you confirm that if untriaged images land on the master branch, and remain untriaged, this post-submit `imgtestAdd` will return `1` for that test?

We have never turned the tree red on post-submit tests, for a number of reasons listed in the linked issue, so I just wanted to check that I was right about the expected return value. :)
Thanks!

Reply:
It will only fail if you have the --passfail [flag](https://github.com/google/skia-buildbot/blob/48428a99b46f7efe47eea0c78a61c213a288c8ab/gold-client/cmd/goldctl/cmd_imgtest.go#L146) set. I don't see that on the imgtest add call, but if it's on the previous imgtest init call, then it would be fine.

Reply:
Ah! Thank you!